src/simulate/utils.py

```diff
@@ -1,12 +1,63 @@
+import datetime
 import logging.config
 import os
+import random
+import uuid
+import string
+import yaml
+from random import randint
 
+PROPS_USER_ID = 'id'
+PROPS_USER_SIS_ID = 'user_sis_id'
+PROPS_USER_CANVAS_ID = 'user_canvas_id'
+PROPS_COURSE_SITE = 'course_site'
+PROPS_COURSE_NAME = 'course_name'
+PROPS_COURSE_SIS_ID = 'course_sis_id'
+PROPS_COURSE_CANVAS_ID = 'course_canvas_id'
+PROPS_VIDEO_PLAYER = 'video_player'
+PROPS_COURSES = 'courses'
+PROPS_USERS = 'users'
+PROPS_HTTP = 'http'
+PROPS_URL = 'url'
 
-def setup_logging(path='config/logging.ini',
+
+def setup_logging(path='config/logging.yml',
                   level=logging.INFO,
                   env_key='LOG_CFG'):
     log_file_path = os.getenv(env_key, path)
     if log_file_path and os.path.exists(log_file_path):
-        logging.config.fileConfig(log_file_path)
+        with open(log_file_path, 'rt') as f:
+            config = yaml.load(f.read())
+            logging.config.dictConfig(config)
     else:
         logging.basicConfig(level=level)
+
+
+def get_uuid():
+    return "urn:uuid:" + str(uuid.uuid4())
+
+
+def get_random_alpha_numeric_string(length):
+    return ''.join(random.choices(string.hexdigits, k=length))
+
+
+def get_current_date_time_iso8601_format():
+    iso_utc_now = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')
+    iso_utc_now = iso_utc_now[:-3] + 'Z'
+    return iso_utc_now
+
+
+def get_current_date():
+    return datetime.datetime.utcnow().strftime('%m/%d/%Y')
```

Comment:
To avoid the repetitive `datetime.datetime` multiple places in your code, import the module this way:
```python
from datetime import datetime
```
Then shorten your code to:
```python
    return datetime.utcnow().strftime('%m/%d/%Y')
```